Changes to fit the requirements of Grants-Program

JimmyChg opened the pull request:
In this PR I've:
* Remove the last variables with myNFT as prefix
* Fix the test
* Add dockerfile + docs on it
* refact contract to have one truffle stack
* add test on manipulator

JimmyChg committed:
chore: remove myNFT from naming convention (part.2)

JimmyChg committed:
fix: update path + fix signing message

JimmyChg committed:
chore: add dockerfile

JimmyChg committed:
chore: clean dockers + docs

JimmyChg committed:
link to internal repo, not github

JimmyChg committed:
Merge branch 'master' into update-jcheng

alex38160 commented on Codebase/Relay/conf.json:
```diff
@@ -1,21 +1,20 @@
         "host": "ipfs.infura.io",
         "port": 5001,
         "protocol": "https",
         "projectId": "205glgJgV59a6lg5A3w9qCWCS8k",
         "projectSecret": "78db432020396e4d0bf8963731a6b17a"
```
Remove projectId & projectSecret values

JimmyChg committed:
remove useless value

JimmyChg committed:
big refactoring

Put the Bridge and Manipulator contract in one folder, to be able to
use them in one Truffle stack

JimmyChg committed:
docs: update docs due to refact

JimmyChg committed:
chore: clean docs, gitignore

JimmyChg committed:
chore: adding the manipulator setup for test

JimmyChg committed:
WIP tests

JimmyChg committed:
chore: differentiate setup for manipulator and bridge_1

create 2 files for setting up the bridge and manipulator:
* one setting up the 2 bridges
* one setting up the 2 bridges and 2 manipulators

JimmyChg committed:
chore: retrieve events data

JimmyChg committed:
fix: fix the call of migrateFromIOUERC721ToERC721

JimmyChg committed:
docs + cleaning

JimmyChg committed:
chore: deploy the IOU contract with the manipulator

JimmyChg committed:
fixing interfaceID of ERC721

JimmyChg committed:
docs

JimmyChg committed:
fixing gasFee price if using ganache
